disperser/batcher/metrics.go

```diff
@@ -59,12 +60,12 @@ type Metrics struct {
 	logger   common.Logger
 }
 
-func NewMetrics(httpPort string, logger common.Logger) *Metrics {
+func NewMetrics(blobMetadataStore *blobstore.BlobMetadataStore, httpPort string, logger common.Logger) *Metrics {
```

Comment:
I'm not sure if adding metadata store as a dependency to Metrics makes sense. 
I was going to suggest we place this probe in dataapi (like how we track non-signers), so that we have observability even when batcher is down. wdyt?

Reply:
Yeah agree, going to move it there